.jshintrc

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
JSHint is bad, ESLint is good. Download the atom plugin `linter-eslint` and you will have the same experience you are used to with JSHint.